Formula/u/util-linux.rb

```diff
@@ -58,17 +57,6 @@ class UtilLinux < Formula
     conflicts_with "rename", because: "both install `rename` binaries"
   end
 
-  # Fix for https://github.com/util-linux/util-linux/issues/3071
-  # Remove with `autoconf` and `automake` build deps when included in a release.
```

Comment:
Removing autoconf/automake was forgotten?

Reply:
thanks! doing this in #177451